core/src/main/scala/sttp/tapir/Schema.scala

```diff
@@ -40,10 +41,12 @@ case class Schema[T](
 
   def show: String = s"schema is $schemaType${if (isOptional) " (optional)" else ""}"
 
-  def modifyUnsafe(fields: String*)(modify: Schema[_] => Schema[_]): Schema[T] = modifyAtPath(fields.toList, modify)
+  def modifyUnsafe[U](fields: String*)(modify: Schema[U] => Schema[U]): Schema[T] = modifyAtPath(fields.toList, modify)
```

Comment:
do we need this `[U]`? It seems it only causes a type parameter ot be required at use-site

Reply:
> It seems it only causes a type parameter ot be required at use-site 
Only while using the unsafe variant, IMO that will be a very rare case.